src/blog/2023/06/flowforge-1-8-released.md

```diff
@@ -0,0 +1,81 @@
+---
+title: FlowForge now offers High Availability Node-RED
+subtitle: FlowForge 1.8 makes Node-RED applications more reliable and scalable, plus more streamline deployment pipelines.
+description: FlowForge now supports High Availibility, DevOps Pipelines and more
+date: 2023-06-08 
+authors: ["marian-demme"]
+image: /blog/2023/06/images/release-1.8.0.png
+---
+
+The FlowForge 1.8 update introduces a suite of features designed to optimize your Node-RED experience. High Availability ensures your business-critical processes stay running under heavy workloads, while DevOps Pipelines allow for smoother, staged deployments. Additionally, the integration of a new User Interface for the Device Agent significantly streamlines the configuration process.
+
+<!--more-->
+
+## More reliable and scalable Node-RED applications
+Do you want to run business-critical processes in Node-RED and need to ensure they are always available and can handle increasing workloads? We're excited to introduce High Availability (HA) in FlowForge to support your needs. Suppose you're managing a large influx of HTTP requests to your Node-RED instances. With our latest update, FlowForge now ensures that these requests are evenly distributed across two instances. This means your applications are less likely to experience downtime and can manage higher traffic loads. You can easily activate HA mode during the creation of an instance. For more details, please see our [documentation](https://flowforge.com/docs/user/).
+
+High Availability is our first [preview feature](https://flowforge.com/handbook/product/versioning/#preview-features), and your feedback is crucial. We encourage you to try out HA in your Node-RED instances and share your experiences with us. Your feedback will help us refine this feature and make it even better.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/zS6P3RR86vE" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+## DevOps Pipelines
+
+DevOps Pipelines enable you to manage and track multiple deployments of your Node-RED instances, effortlessly progressing through stages such as testing, development, staging, and production. In [FlowForge Version 1.4](../../02/flowforge-1-4-0-released),  we introduced the "Support for Staged Development". Now, we're taking it one step further. Within your applications section, you are now able to create your own DevOps Pipeline and track the progress across each instance in one view.
```

Comment:
I would suggest rewording as follows:

FlowForge 1.8 introduces the concept of Pipelines to better organize your Node-RED development. Development team can now setup different staging environments for different steps in the development cycle, ex. test, development and production. Node-RED instances can be pushed along a pipeline as they move along the development process. This allows for a better organized and predictable development process for your team.

The new Pipelines feature builds upon the Staged Development support that we introduced in 1.4.

Reply:
```suggestion
FlowForge 1.8 introduces the concept of Pipelines to better organize your Node-RED development. Development team can now setup different staging environments for different steps in the development cycle, ex. test, development and production. Node-RED instances can be pushed along a pipeline as they move along the development process. This allows for a better organized and predictable development process for your team. 

The new Pipelines feature builds upon the Staged Development support that we introduced in[FlowForge Version 1.4](../../02/flowforge-1-4-0-released).
```

Reply:
@MarianRaphael I think that's much better already, can we add a line that explains they no longer should develop their flows in production instances? While you implicitly touch on it, it might be good to make it explicit.

Reply:
```suggestion
FlowForge 1.8 introduces the concept of Pipelines to better organize your Node-RED development. Development team can now set up different staging environments for different steps in the development cycle, ex. test, development and production. Node-RED instances can be pushed along a pipeline as they move along the development process. This allows for a better organized and predictable development process for your team. 

The new Pipelines feature builds upon the Staged Development support that we introduced in[FlowForge Version 1.4](../../02/flowforge-1-4-0-released). We highly recommend that development teams avoid developing their flows directly in production instances. This approach fosters a more reliable and robust development process, reducing the risks associated with production environment modifications. Instead, start your development in a dedicated development or test instance and then promote your Node-RED instances to production once they have been thoroughly tested and reviewed. 
```